elastic/apm-server
# Introduce session.{id,sequence} for transactions

## Motivation/summary

Introduce `session.id` and `session.sequence` fields to transactions for RUM.

## Checklist


## How to test these changes

Using https://github.com/elastic/apm-agent-rum-js/pull/1005, capture some transactions that include `session.id` and `session.sequence` and ensure the fields are recorded in Elasticsearch with the expected mappings (`keyword` and `long`).

## Related issues

Closes #5032
